site/content/en/docs/admission-check-controllers/multikueue.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "MultiKueue Admission Check Controller"
+date: 2023-10-23
+weight: 2
+description: >
+  An admission check controller providing the manager side MultiKueue functionality.
+---
+
+The MultiKueue Admission Check Controller is an Admission Check Controller designed to provide the manager side functionality for [MultiKueue](https://github.com/kubernetes-sigs/kueue/blob/main/keps/693-multikueue/README.md). 
+
+The controller is part of kueue. You can enable it by setting the `MultiKueue` feature gate. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
+
+{{% alert title="Warning" color="warning" %}}
+MultiKueue is currently an alpha feature and disabled by default.
+{{% /alert %}}
+
+The controller's main attributes are:
```

Comment:
Maybe we should have something more user / admin - oriented. I'm thinking, what would be particularly useful:
1  (admin pov) demonstrate step-by-step how admin would setup the clusters (installing jobset etc. on the workers, but only CRDs), with example configs for the manager and worker clusters 
2. (user pov) explain how user can interact with the environment, explaining that users interacts only with the manager cluster, but should not expect the pods to be created there. If they are interested in debugging the job they need to figure out the worker the job is actually running on, connect to the cluster and see pods.

This could probably be follow ups for the admin / user tasks. WDYT @tenzen-y @alculquicondor @mwielgus ?

Reply:
> This could probably be follow ups for the admin / user tasks. WDYT @tenzen-y @alculquicondor @mwielgus ?

That makes sense. 
Also, I think we should clarify the responsibilities of each role (user, admin, kueue-controller-manager), like kubeconfig, in the `task` section.

Reply:
Indeed, the current paragraph contains implementation details.

We need to briefly start with the architecture (manager, workers) and when jobs are sent to the workers (first they need to pass quota checks in the manager, then the manager makes the worker clusters compete to run the job).

We should also mention that the manager cluster should have a quota roughly equal to the total quota among all clusters.